repo/maintenance/blob_retain_test.go

```diff
@@ -85,8 +84,6 @@ func (s *formatSpecificTestSuite) TestExtendBlobRetentionTimeDisabled(t *testing
 			nro.BlockFormat.MasterKey = testMasterKey
 			nro.BlockFormat.Hash = blockFormatHash
 			nro.BlockFormat.HMACSecret = testHMACSecret
-			nro.RetentionMode = blob.Governance
```

Comment:
why this change?

Reply:
Governance without retention time is not really a useful configuration, and irrelevant to what is being tested here.  The main purpose of this test is to verify that things work ok if retention is disabled (even if the maintenance parameter is enabled).  The important thing is that `blobCfg.IsRetentionEnabled()` returns false, and this test is not intended to validate the functionality of that method.